example/speech_command/Makefile

```diff
@@ -3,3 +3,12 @@ train:
 	mkdir -p models data
 	python3 sc/train.py
 
+.POHNY: prepare-e2e-data
+prepare-e2e-data:
+	[ -d data ] ||  mkdir -p data
+	[ -f data/speech_commands_v0.02.tar.gz ] || curl -o data/speech_commands_v0.02.tar.gz http://10.131.0.1:9000/users/dataset/SpeechCommands/speech_commands_v0.02.tar.gz
```

Comment:
aliyun oss for public is better choice than `10.131.0.1`? 

@xuchuan  we need an OSS bucket that can provide the public HTTP download service.

Reply:
done